### DISCUSSION TITLE
How can tailwindcss co-worker with UED(UI/UX) team, is there any plugin can use it?

### DISCUSSION THREAD

**Discussion by pro-free-dev** [tailwindlabs/tailwindcss#11651] [Help]
hi team:
I'm a front-end developer and designer, how can use sketch to output tailwindcss configurations?

**Comment by ch00z00** [comment]
If a designer is using Figma, the [TailwindCSS plugin](https://www.figma.com/community/plugin/738806869514947558/Tailwind-CSS) is useful.

Also, if you want to streamline coding from Figma design comps, [Figma to Code](https://www.figma.com/community/plugin/842128343887142055/Figma-to-Code-(HTML,-Tailwind,-Flutter,-SwiftUI)/Figma-to-Code-(HTML%2C-Tailwind%2C-Flutter%2C-SwiftUI)) is recommended.
